cmd/uplink/cmd/setup.go

```diff
@@ -20,13 +20,29 @@ import (
 )
 
 var (
-	setupCmd = &cobra.Command{
+	uplinkSetupCmd = &cobra.Command{
 		Use:         "setup",
 		Short:       "Create an uplink config file",
-		RunE:        cmdSetup,
+		RunE:        cmdUplinkSetup,
 		Annotations: map[string]string{"type": "setup"},
 	}
-	setupCfg struct {
+	gatewaySetupCmd = &cobra.Command{
```

Comment:
How about moving all things related to gateway under `/cmd/gateway/cmd`?

This we would also avoid complicating var/type names to distinguish if they relate to the uplink or the gateway.

Reply:
Yea, good idea